holopy/inference/prior.py

```diff
@@ -83,6 +84,19 @@ def unscale(self, scaled):
 
 class Uniform(Prior):
     def __init__(self, lower_bound, upper_bound, guess=None, name=None):
+        """
+        Uniform prior.
+
+        Parameters
+        ----------
+        lower_bouund, upper_bound : float
+        guess : float or None, optional
+            The value to take as an initial guess from the prior. If
+            guess is None, defaults to the midpoint of the prior for
+            most cases.
```

Comment:
most cases means non-improper (proper?) priors. It might be good to clarify that.

Reply:
Should be fixed by 68438b8